.github/workflows/build-api.yaml

```diff
@@ -31,5 +32,6 @@ jobs:
           image: "us-docker.pkg.dev/integrationos/docker-oss/api:${{ env.docker_image_tag }}"
           service_account: github-actions@integrationos.iam.gserviceaccount.com
           workload_identity_provider: projects/356173785332/locations/global/workloadIdentityPools/github-actions/providers/github-actions
+          file: packager/Dockerfile.api
```

Comment:
Consider moving these Dockerfiles so that they live with the service that they're building , i.e. `integrationos-api/Dockerfile`

Reply:
Will do!